File: Borealis.Api/app/dto/pfo_collection_dto.py
class PFOCollectionDto():
    def __init__(self, page, page_count, per_page, order_by, order_by_descending, items):
        self.page = page
        self.page_count = page_count
        self.per_page = per_page
        self.order_by = order_by
        self.order_by_descending = order_by_descending
        self.items = items

File: Borealis.Web/src/app/dtos/pfoCollectionDto.ts

export class PFOCollectionDto<T>{
    public page: Number;
    public pageCount: Number;
    public perPage: Number;
    public orderBy: String;
    public orderByDescending: Boolean;
    public items: T[];

    constructor(page, pageCount, perPage, orderBy, orderByDescending, items) {
        this.page = page;
        this.pageCount = pageCount;
        this.perPage = perPage;
        this.orderBy = orderBy;
        this.orderByDescending = orderByDescending;
        this.items = items;
    }
}